stylesheet.css

```diff
@@ -0,0 +1,11 @@
+
+img 
+{ object-fit: fill;
+
+    width = "100px";
+    height = "100px";
+    border = "0px";

```

Comment:
CSS has a different syntax. Check the properties syntax here https://www.w3schools.com/css/